acspy/acsc.py

```diff
@@ -333,7 +339,13 @@ def getFPosition(hcomm, axis: int, wait=SYNCHRONOUS):
     call_acsc(acs.acsc_GetFPosition, hcomm, axis, byref(fposition), wait)
     fposition = fposition.value
     return fposition
-
+    
+def getTPosition(hcomm, axis: int, wait=SYNCHRONOUS):
+    """Retrieves an instant value of the motor target position."""
+    tposition = ctypes.c_double()
+    call_acsc(acs.acsc_GetTargetPosition, hcomm, axis, byref(tposition), wait)
+    tposition = tposition.value
+    return tposition
```

Comment:
Why use a `ctypes.c_double` here and a `double` in the other function?

```suggestion


def getTPosition(hcomm, axis: int, wait=SYNCHRONOUS):
    """Retrieves an instant value of the motor target position."""
    tposition = ctypes.c_double()
    call_acsc(acs.acsc_GetTargetPosition, hcomm, axis, byref(tposition), wait)
    tposition = tposition.value
    return tposition

```

Reply:
To be honest I have not specific reasons, I didn't want to miss up the style so I copy getFPosition function and change the acsc command 
but i think in jerk position i forgot to add ctypes 
sorry for the late replay